### PR TITLE
Release v1.8.5 Develop branch to Main

### PR DESCRIPTION
**Pull request by sgiavasis**
## Fixes

Related to [v1.8.5 release](https://github.com/FCP-INDI/C-PAC/milestone/24)

## Description

Merges the `develop` brain to `main` for the final release of v1.8.5.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the `main` branch of the repository. 
- [x] My commit messages follow best practices.
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [x] I updated the changelog.
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: dev/docker_data/run.py [hunk 1]
```diff
@@ -251,11 +248,14 @@ def run_main():
     parser.add_argument('--save_working_dir', nargs='?',
                         help='Save the contents of the working directory.',
                         default=False)
-    parser.add_argument('--disable_file_logging', action='store_true',
-                        help='Disable file logging, this is useful for '
-                             'clusters that have disabled file locking.',
-                        default=False)
+    parser.add_argument('--save_workflow',
+                        help='Save a serialized version of the workflow. '
+                             'Can be used with `test_config` save workflow '
```
Comment (shnizzedy) [inline dev/docker_data/run.py:253]:
```suggestion
                             'Can be used with `test_config` to save workflow '
```

File: CPAC/info.py [hunk 2]
```diff
@@ -43,8 +43,8 @@
 # version
 _version_major = 1
 _version_minor = 8
-_version_micro = 4
-_version_extra = ''
+_version_micro = 5
+_version_extra = 'dev1'
```
Comment (shnizzedy) [inline CPAC/info.py:47]:
```suggestion
_version_extra = ''
```

**Review by shnizzedy — COMMENTED** [review]
Idk if it's better to mark the version non-dev before or after we merge into [`main`](https://github.com/FCP-INDI/C-PAC/tree/main), but it seems like making the change here might be better practice to minimize commits directly in [`main`](https://github.com/FCP-INDI/C-PAC/tree/main).

Also, opened a PR (https://github.com/FCP-INDI/C-PAC/pull/1892) to remove a couple empty files before we merge

File: CPAC/seg_preproc/seg_preproc.py [hunk 3]
```diff
@@ -907,27 +907,70 @@ def tissue_seg_freesurfer(wf, cfg, strat_pool, pipe_num, opt=None):
...
+        outputs['label-GM_mask'] = (erode_tissues['gm'], 'binary_file')
+
+    else:
+        outputs['label-CSF_mask'] = (pick_tissue, 'csf_mask')
+        outputs['label-WM_mask'] = (pick_tissue, 'wm_mask')
+        outputs['label-GM_mask'] = (pick_tissue, 'gm_mask')
+
     fs_seg, hemisphere_outputs = create_seg_preproc_freesurfer(
         config=cfg, wf_name=f'seg_preproc_freesurfer_{pipe_num}',
         pipe_num=pipe_num)
```
[the remark this answers is not in the document]
Comment (sgiavasis) [reply CPAC/seg_preproc/seg_preproc.py:976]: The commenting out/removal of that older function was intentional, but the usage within the nodeblock wasn't.

@e-kenneally is on it.

**Review by shnizzedy — APPROVED** [review]

**Pull request merged**